docs/configuration_options.md

```diff
@@ -0,0 +1,303 @@
+# Configuration Reference
+
+TODO: SECTIONS(DELAYED JOB SUPPORT, ASYNC SUPPORT, WRITE TO FILE, PERSON TRACKING)
+
+<!-- Sub:[TOC] -->
+
+## access_token
+
+**Required**
+
+Sets the access token used to send payloads to rollbar.
+
+Items sent through a given access token arrive in that access token's project
+and respect the rate limits set on that access token.
+
+## async_handler
+
+If `config.use_async = true` explicitly sets the function used to send
```

Comment:
The object passed to this option should respond to `#call` and receive the payload. Examples in `lib/rollbar/delay/`.